scripts/analysis.py

```diff
@@ -866,48 +837,48 @@ def capacity_calc(governments, timestep, entry, exit_step):
 
     Returns
     -------
-    tuple
-        (power_dict, num_dict) which holds timeseries
-        of capacity and number of reactors
-        with country_government as key
+    power_dict: dictionary
```

Comment:
Either return just one dictionary (which stores both power and deployment), or split this into two functions.